### ISSUE TITLE
Use --use-json-log for rclone to reduce complexity

### ISSUE THREAD

**Issue by newhinton** [x0b/rcx#156]
#### Pre-Submission checklist

 - [x] There is no existing issue that already asks for this feature.<br />
 - [x] The feature already exists in rclone (on your PC or in Termux)
 - [x] I am prepared to help make this feature

#### What problem are you trying to solve?
Currently rcx parses the human readable output of rclone to display it's notification. It works, but it has a huge maintenance overhead. Also it is not easily possible to aggregate the information to a final summary.

#### What should RCX be able to do differently to help with this problem?
rcx should switch to use --use-json-log which contains all the same information as before, except it is in an easily readable machine format. this allows to expand the usefulness of notifications while also reducing complexity.

I am willing to work on this after #84 is done to support #155

**Comment by newhinton** [comment]
implemented in #155

**Comment by x0b** [comment]
Seems reasonable. Can you factor this into its own PR?  Not sure about the summary info though - isn't that provided by rclone itself?

FYI: I'm planning to move RCX from command-line based process invocations to a shared libary model (#130). File transfer status information will be routed to [RcdService](https://github.com/x0b/rcx/blob/fbb5ad39f241b198f078da36366b101e8baac05d/app/src/main/java/ca/pkay/rcloneexplorer/Services/RcdService.java), which provides the foreground priority (i.e. should not be killed by Android). Once that is ready, all other transfer services will be removed. Just that you know and don't spend too much effort on improving the old components.

**Comment by newhinton** [comment]
yes the summary is provided by rclone, which is problematic since it is untranslatable. the flag provides all the info the human readable output provides, except it is nicely machine readable and can therefore be translated, changed and modified at our will.

**Comment by newhinton** [comment]
I have recreated the pr without the logging feature. 
#160 

Without any further additions, it does not add anything visible, it should however make future adjustments more easy.
